dicom2/loadDirectory.py

```diff
@@ -39,6 +40,7 @@ def loadDirectory(path, patientID=None, studyID=None, seriesID=None):
                 patient = patients[DCMImage.PatientID]
             else:
                 patient = patients.add(DCMImage)
+               
```

Comment:
We might want to look into code formatting for you. In PyCharm, I code format it a certain way to be consistent.

One blank line between code unless it is between two functions, then it is two blank lines.

Remove the blank line here and above

Reply:
I agree, I'd like to stay consistent with what you already have in place. Makes it easier to read the code as a whole. I'll also start using camelCase as mentioned below